flink-python/src/main/java/org/apache/flink/streaming/api/runners/python/beam/BeamPythonFunctionRunner.java

```diff
@@ -543,12 +543,14 @@ private ExecutableStage createExecutableStage(RunnerApi.Environment environment)
     private static StateRequestHandler getStateRequestHandler(
             KeyedStateBackend keyedStateBackend,
             TypeSerializer keySerializer,
+            TypeSerializer windowSerializer,
```

Comment:
If we decide to rename `window_coder` to `namespace_coder` at python side, we also need to rename the `windowSerializer` here.

Reply:
Make sense.